proto/neutron/cron/schedule.proto

```diff
@@ -5,6 +5,13 @@ import "gogoproto/gogo.proto";
 
 option go_package = "github.com/neutron-org/neutron/v4/x/cron/types";
 
+// BlockerType defines when messages will be executed in the block
+enum BlockerType {
```

Comment:
Need to think of a better name. How about:
```go
enum ExecutionStage {
  BEGIN_BLOCKER = 0;
  END_BLOCKER = 1;
  BOTH_BLOCKERS = 2;
}
```

Reply:
Done